topo/node/node.go

```diff
@@ -122,6 +135,35 @@ func (n *Impl) GetNamespace() string {
 	return n.Namespace
 }
 
+func (n *Impl) TopologySpecs(context.Context) ([]*topologyv1.Topology, error) {
+	proto := n.GetProto()
+
+	t := topologyv1.Topology{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: proto.Name,
+		},
+		Spec: topologyv1.TopologySpec{
+			Links: make([]topologyv1.Link, len(proto.Interfaces)),
+		},
+	}
+
+	i := 0
```

Comment:
i would change this just slightly

```go

var links []topologyv1.Link
for ifcName, ifc := range proto.Interfaces {
  links = append(links, topologyv1.Link{
    UID:  int(ifc.Uid),
    LocalIntf: ifcName,
    PeerIntf: ifc.PeerIntfName,
    PeerPod: ifc.PeerName,
  })
}

return []*topologyv1.Topology{&topologyv1.Topology{
  		ObjectMeta: metav1.ObjectMeta{
			Name: proto.Name,
		},
		Spec: topologyv1.TopologySpec{
			Links: links,
		},
}}
```

Reply:
https://github.com/google/kne/commit/db2ba56de37628c405180ae609ceb6ee706c4011